infra/vaulted.tfvars

```diff
@@ -0,0 +1,6 @@
+$ANSIBLE_VAULT;1.1;AES256
```

Comment:
Could you add to the README how the passwords are encrypted  and how to use that ?

Reply:
I can but Ill not let the password in the README. I'm on it.